src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Random randomIndex = new Random();
+    private final int randomIndexInt = randomIndex.nextInt(Colors.values().length);
```

Comment:
this number should be local - it should be created each time the method is invoked